tzwhere/tzwhere.py

```diff
@@ -48,6 +57,8 @@ def __init__(self, input_kind='json', path=None):
             pgen = tzwhere._feature_collection_polygons(featureCollection)
         elif input_kind == 'csv':
             pgen = tzwhere._read_polygons_from_csv(path)
+        elif input_kind == 'csv_gz':
+            pgen = tzwhere._read_polygons_from_csv_gz(path)
```

Comment:
Oops, good find. These two lines are to be removed as it is now merged with `csv`.